pxf/InstallPXFPlugins.html.md.erb

```diff
@@ -27,7 +27,7 @@ If you are using Ambari to install and manage your HAWQ cluster, you do *not* ne
 
 Each PXF service plug-in resides in its own RPM.  You may have built these RPMs in the Apache HAWQ open source project repository (see [PXF Build Instructions](https://github.com/apache/incubator-hawq/blob/master/pxf/README.md)), or these RPMs may have been included in a commercial product download package.
 
-PXF software must be installed on *each* node in your cluster.
+Perform the following steps to install PXF software on **_each_** node in your cluster.
```

Comment:
I think for clarity this should be reversed:  Perform the following steps on **_each_** node in your cluster to install PXF.